### ISSUE TITLE
Editera belastning

### ISSUE THREAD

**Issue by fsergler** [JohanFalt/Foundry_EON-RPG#228]
Finns regler för lättare rustning, vilket säker belastningen med X antal nivåer.

**Comment by JohanFalt** [comment]
Detta är från Strid?

**Comment by fsergler** [comment]
Nej, det finns händelse slag som gör att rustningar kan vara lättare. Samt så har dvärgar denna egenskap: 

> Kort och kraftig: En zolod kan bära tunga rustningar utan att hindras. De modifierar sin totala Belastning med –8 för att se hur belastade de är.

Så denna ändring gör det främst lättare att hantera belastningen för dvärgar.
